components/style/dom.rs

```diff
@@ -877,6 +877,12 @@ pub trait TElement:
         hints: &mut V,
     ) where
         V: Push<ApplicableDeclarationBlock>;
+
+    /// Returns element's local name.
+    fn local_name(&self) -> &LocalName;
```

Comment:
This still needs to use the `Borrowed*` gunk, so that you don't break Firefox.

Also this actually needs to be implemented, right?

Reply:
Not sure what exact type should be used?

Yes. It's used in the code.